crates/runtime/src/traphandlers.rs

```diff
@@ -70,6 +71,9 @@ cfg_if::cfg_if! {
                 register(&mut PREV_SIGFPE, libc::SIGFPE);
             }
 
+            // on ARM64, we use `brk` to report traps, which generates SIGTRAP.
+            register(&mut PREV_SIGTRAP, libc::SIGTRAP);
```

Comment:
Could you put this under a `cfg!(target_arch = "aarch64")` so that we don't register handlers for things we don't need to on x86?

Reply:
Actually, a few days ago we found an alternative opcode that does deliver `SIGILL` on arm64, and I hadn't updated this yet -- so we can just remove it.